client/resource_feature_store.go

```diff
@@ -169,6 +169,11 @@ func folderDestinationSchema() *schema.Resource {
 				Type:     schema.TypeBool,
 				Required: true,
 			},
+			"save_mode": {
+            		    Type:     schema.TypeString,
+            		    Required: true,
+            		    ValidateFunc: validation.StringIsNotWhiteSpace,
```

Comment:
This should validate that it's in a slice of correct values. See aggregation type in feature.

Reply:
done